_docs/style/colors.md

```diff
@@ -60,32 +192,162 @@ illustrations. They should never overpower the primary colors.
         {%endfor%}
       </div>
     {%endfor%}
-  </div>
+</div>
+{% endcolumn %}
+
+</div>
+
 </section>
 
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+You can also use status labels as a way to communicate the status of an object, for example, a support ticket. Unlike health status that prescribes that you use specific icons and colors to convey meaning, the object status family offers more freedom. You can choose from among the colors to attract the right amount of attention to the status of an object.
+
+Review your system of statuses to determine which label to use.
+
+- Use gray 400 for a subdued status label.
+- Use gray 650 as the default status label.
+- Use gray 800 to communicate emphasis.
+- Use indigo 800 to communicate extra emphasis.
+
+Refer to Status Labels for more detailed usage and construction guidelines.
```

Comment:
**Status Labels** (or make this a link to the information like Interactive States below)

Reply:
I commented this line out. That should be a xref to another component page called Status Labels.